sidebarsDocs.js

```diff
@@ -23,528 +23,652 @@ const sidebars = {
-      label: 'Protocol',
-      link: {type: 'doc', id: 'protocol'},
-      items: [
-        {
-          type: 'doc',
-          id: 'specification',
-          label: 'Compatibility',
-        },
-        'interoperability',
-        'semantics',
-        {
-          type: 'doc',
-          id: 'amqp-0-9-1-quickref',
-          label: 'Quick Reference',
-        },
-        {
-          type: 'doc',
-          id: 'amqp-0-9-1-reference',
-          label: 'Full Reference',
-        },
-        {
-          type: 'doc',
-          id: 'amqp-0-8-to-0-9-1',
-          label: 'Differences from AMQP 0-8 to 0-9-1',
-        },
-        {
-          type: 'doc',
-          id: 'amqp-wireshark',
-          label: 'Traffic Analyzer',
+          id: 'reliability',
+          label: 'Reliable Delivery of Messages and Data Safety',
+        },
+        {
+        type: 'category',
+        label: 'Publishing Messages',
+        items: [
+          {
+            type: 'doc',
+            id: 'e2e',
+            label: 'Exchange to Exchange Bindings',
+          },
+          {
+            type: 'doc',
+            id: 'ae',
+            label: 'Alternative Exchanges',
+          },
+          {
+            type: 'doc',
+            id: 'sender-selected',
+            label: 'Sender-selected Distribution',
+          },
+        ],
+      },
+      {
+        type: 'category',
+        label: 'Consuming Messages',
+        link: {type: 'doc', id: 'admin-guide'},
+        items: [
+          {
+            type: 'doc',
+            id: 'consumers',
+            label: 'Describing Consumers',
+          },
+          {
+            type: 'doc',
+            id: 'consumer-cancel',
+            label: 'Consumer Cancellation Notifications',
+          },
+          {
+            type: 'doc',
+            id: 'consumer-prefetch',
+            label: 'Consumer Prefetch',
+          },
+          {
+            type: 'doc',
+            id: 'consumer-priority',
+            label: 'Consumer Priorites',
+          },
+        ],
+      },
+      {
+        type: 'doc',
+        id: 'confirms',
+        label: 'Consumer Acknowledgements and Publisher Confirms',
+      },
+      {
+        type: 'category',
+        label: 'Working with High Availability Queues and Streams',
+        items: [
+          {
+            type: 'doc',
+            id: 'queues',
+            label: 'Queues in General',
+          },
+          {
+            type: 'doc',
+            id: 'quorum-queues',
+            label: 'Quorum Queues',
+          },
+          {
+            type: 'doc',
+            id: 'classic-queues',
+            label: 'Classic Queues',
+          },
+          {
+            type: 'doc',
+            id: 'classic-queues',
+            label: 'Migrate Mirrored Classic Queues to Quorum Queues',
+          },
+          {
+            type: 'doc',
+            id: 'ttl',
+            label: 'Queue and Message TTL and the Expiration Feature',
```

Comment:
We need a shorter label.

Reply:
Changed label to "Time-to-Live and Expiration", this title is used in the existing site